proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10532,6 +10532,15 @@ function bufferRetype( test )
   test.shouldThrowErrorSync( () => _.bufferRetype( new BufferRaw( 5 ) ), I8x, 'extra' );
 }
 
+function bufferRetypeWithOffset( test )
+{
+  test.case = 'F64x from BufferRaw with offset to U16x';
+  var raw = new BufferRaw( 32 );
+  var src = new F64x( raw, 0, 4 );
+  var got = _.bufferRetype( src, U16x );
+  var expected = new U16x( [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ] );
+  test.identical( got, expected );
```

Comment:
Please, implement next combination of src and bufferTyped
| src                 |  buffertyped                    |
|-------------------|----------------------------------|
|  BufferNode   | U8x, I16x, F32x, U64x   |
|  BufferView    | U8x, I16x, F32x, U64x   |
|  I8x                |  U8x, U16x, F32x, I64x  |
|  I32x              |  U8x, U16x, F32x, I64x  |
|  I64x              |  U8x, U16x, F32x, U64x  |
|  F32x             |  U8x, U16x, I32x, I64x  |